README.md

```diff
@@ -73,6 +73,7 @@ programs.neovim.plugins = [
 - https://github.com/nvim-telescope/telescope-frecency.nvim
 - https://github.com/kkharji/lispdocs.nvim
 - https://github.com/nvim-telescope/telescope-cheat.nvim
+- https://github.com/mrjones2014/legendary.nvim
```

Comment:
let me know if you'd like me to remove this and put it in a separate PR 🙂

Reply:
Feel like you already know the answer, Indeed that would be better 😬